api/requests.js

```diff
@@ -1,4 +1,6 @@
-const baseUrl = 'https://coronahelfer.eu/api/v1'
+const baseUrl = process.env.NODE_ENV === 'production'
+  ? 'https://coronahelfer.eu/api/v1'
+  : 'http://localhost:3000/api/v1'
```

Comment:
What is with dev?

Reply:
what you call dev is actually staging. but yes I should account for that as well

Reply:
Yes, we call the staging here Dev ;)

Reply:
for the staging environment there is not yet a backend. the production backend is used. so this should be fine